doc/source/about/team.md

```diff
@@ -1,9 +1,18 @@
 # Team
 
 ## Core developers
+- Joris Van den Bossche - **maintainer** | [@jorisvandenbossche](https://github.com/jorisvandenbossche)
+- Martin Fleischmann | [@martinfleis](https://github.com/martinfleis)
+- James McBride | [@jdmcbr](https://github.com/jdmcbr)
+- Brendan Ward | [@brendan-ward](https://github.com/brendan-ward)
+- Levi Wolf | [@ljwolf](https://github.com/ljwolf)
 
-- Joris Van den Bossche
-- Martin Fleischmann
-- James McBride
-- Brendan Ward
-- Levi Wolf
+## Founder
+
+- Kelsey Jordahl | [@kjordahl](https://github.com/kjordahl)
+
+## Alumni developers
```

Comment:
@jorisvandenbossche, @jdmcbr do you know who else should be listed here?

Reply:
We should probably look at the people in the github team (but maybe not all listed there should be listed here). I can do a "best guess" and then we should maybe ping them to check if they are OK with being listed here.

Reply:
>  I can do a "best guess"

Can you give it a go?